Signal/src/Models/OWSAvatarBuilder.m

```diff
@@ -13,10 +14,15 @@ @implementation OWSAvatarBuilder
 
 + (UIImage *)buildImageForThread:(TSThread *)thread contactsManager:(OWSContactsManager *)contactsManager
 {
+    const CGFloat kDefaultAvatarDiameter = 100;
+    return [self buildImageForThread:thread contactsManager:contactsManager diameter:kDefaultAvatarDiameter];
```

Comment:
You do not have to address this in this PR, but:

A) We should unify this "avatar" sizing with the "custom avatar" sizing at 210px.

See:

* iOS https://github.com/WhisperSystems/Signal-iOS/blob/5b9978b07fcaef170d0ae09dc6f2ae94e2b4450e/Signal/src/ViewControllers/GroupViewHelper.m#L121
* Android https://github.com/WhisperSystems/Signal-Android/blob/4112f23f335ca7fe767edae52bb3b264d9e51c92/src/org/thoughtcrime/securesms/GroupCreateActivity.java#L105

B) _Or_ we should make sure that these avatar sizes reflect the actual view size _in pixels not points_.  In `ContactTableViewCell`, these are shown at 40pt, so 40, 80, 120px. I'm not sure about `InboxTableViewCell`, etc.  But there would be a real quality advantage to rendering to the exact view size.